Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -51,7 +52,7 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         tableView.registerClass(ActionOverlayTableViewCell.self, forCellReuseIdentifier: "ActionOverlayTableViewCell")
         tableView.registerClass(ActionOverlayTableViewHeader.self, forHeaderFooterViewReuseIdentifier: "ActionOverlayTableViewHeader")
         tableView.backgroundColor = UIConstants.PanelBackgroundColor
-        tableView.scrollEnabled = false
+        tableView.scrollEnabled = true
```

Comment:
I'd also set bounces to be false since it looks kind of weird that the header cell can be dragged down!